sshd-core/src/main/java/org/apache/sshd/certificate/OpenSshCertificateBuilder.java

```diff
@@ -267,4 +275,38 @@ public OpenSshCertificate sign(KeyPair caKeypair, String signatureAlgorithm) thr
 
         return cert;
     }
+
+    /**
+     * Validate if certificate options are correct
+     *
+     * @param options Options
+     */
+    private void validateOptions(List<OpenSshCertificate.CertificateOption> options) {
+        if (options != null && !options.isEmpty()) {
+            // check if any duplicates
+            Set<String> names = new HashSet<>();
+            Set<String> duplicates = options.stream().filter(option -> !names.add(option.getName()))
+                    .map(option -> option.getName())
+                    .collect(Collectors.toSet());
+            if (!duplicates.isEmpty()) {
+                throw new IllegalArgumentException("Duplicate option: " + String.join(",", duplicates));
```

Comment:
`Set.toString()` will enumerate the content. So perhaps just `+ duplicates` instead of the `String.join(...)`?

Reply:
Done